openapi.yaml

```diff
@@ -1280,6 +1282,8 @@ paths:
         - BearerAuth: []
       tags:
         - idp-group-mappings
+      x-speakeasy-name-override: update
```

Comment:
can you test if this name override actually does anything? im not convinced it matters. i know we do it elsewhere but i think it was unneeded. I believe it just changes the operationID to a different string which just changes the name of the codegened function but it doesnt actually matter.

if it doesn't would rec just removing it so our terraform overlay file is simpler.

Reply:
sure, I'll test that and lyk